CVLByExample/AddressFunctionCall/AddressCall.spec

```diff
@@ -0,0 +1,10 @@
+using A as a;
+using B as b;
+
+rule AddressCall {
+    env e;
+    address x;
+    require x == a || x == b;
+    assert x == a => x.balaceOf(e) == 1;
+    assert x == b => x.balaceOf(e) == 2;
+}
```

Comment:
1. change balanceOf (since it has a very specific meaning') to 'foo' 
2. and the whole idea is that you don't need 'x==b...` 
so maybe 
`assert x.foo() >= 1 && x.foo <=2`

what if we don't have ` require x == a || x == b;` will it havoc? 
can we show another rule without limiting x 

can you show another rule with

Reply:
Done